README.md

```diff
@@ -1,101 +1,134 @@
     - Each of your spec files should `require_relative` the spec helper file.
 1. Create a test to check the instantiation of one of your object types (**RED**)
 1. Create the class for the object tested in the step above (**GREEN**)
-1. Use git add, commit and push commands to push your initial code to GitHub
+1. Use `git add`, `commit` and `push` commands to push your initial code to GitHub
 
 ### Process
-You should use the following process as much as possible:  
+
+You should use the following process as much as possible:
 
 1. Write pseudocode
 1. Write test(s)
 1. Write code
+1. Commit
+
+You should have **95% code coverage** using `simplecov`.
+
+Your git commit history should provide a clear description of how your code developed, letting the reader know what changed when and why.
+
+## Wave Zero: Project Design
 
-Since this is the **last** Ruby-only project we are doing, and a level 3, the requirements are larger than previous projects. **It is possible you will not be able to complete all requirements**, but if you use the process above, you will still be able to share with your instructors the process and the design you have created.
+This will be an in-class activity.
 
+## Wave One: Tracking Reservations
 
-## Wave One: Initial System Configuration and Setup
+Remember that your job is only to build the classes that store information and handle business logic, and the tests to verify they're behaving as expected. Building a user interface is not part of this project!
 
 ### User Stories
-- As an administrator, I should be able to access the list of all of the rooms in the hotel
-- As an administrator, I should be able to reserve any room for a given date range
-- As an administrator, I should be able to access the list of reservations for a specific date
-- As an administrator, I should be able to get the total cost for a given reservation
 
-### Assumptions
-- There are 20 rooms to start with, and they are numbered 1 through 20
-- There is a single room rate of $200/night to start with
-- Any room can be reserved at any time (do not worry about whether or not it is available - that's in wave 2!)
+- As an administrator, I can access the list of all of the rooms in the hotel
+- As an administrator, I can reserve a room for a given date range
+- As an administrator, I can access the list of reservations for a specific date
+- As an administrator, I can get the total cost for a given reservation
+
+### Constraints
+
+- The hotel has 20 rooms, and they are numbered 1 through 20
+- Every room is identical, and a room always costs $200/night
+- The last day of a reservation is the checkout day, so the guest should not be charged for that night
+- For this wave, any room can be reserved at any time, and you don't need to check whether reservations conflict with each other (this will come in wave 2!)
+
+### Error Handling
 
-### Error Handling Recommendations
 - Your code shall raise an error when an invalid date range is provided
```

Comment:
small comment: "shall" comes across as awkward, consider using "should" or "will"